src/main/java/duke/Duke.java

```diff
@@ -0,0 +1,225 @@
+package duke;
+
+import duke.task.Deadline;
+import duke.task.Event;
+import duke.task.Task;
+import duke.task.ToDo;
+
+import java.io.File;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.Scanner;
+import java.util.ArrayList;
+
+public class Duke {
```

Comment:
There are hardly any comments in your code. Perhaps you can try adding method headers (See [javadocs](https://se-education.org/guides/conventions/java/basic.html#comments) to briefly explain what the methods are doing.